tt_eager/tt_dnn/op_library/prod/kernels/compute/prod_all.cpp

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include "compute_kernel_api/common.h"
+#include "compute_kernel_api/tile_move_copy.h"
+#include "compute_kernel_api/eltwise_unary/eltwise_unary.h"
+#include "tt_metal/include/compute_kernel_api/eltwise_binary.h"
+#include "compute_kernel_api/eltwise_unary/sfpu_split_includes.h"
+#include "compute_kernel_api/eltwise_unary/negative.h"
+
+ALWI void ACQ() {
+    tile_regs_acquire();
+    tile_regs_wait();
+}
+ALWI void REL() {
+    tile_regs_commit();
+    tile_regs_release();
+}
```

Comment:
This is basically just replicating acqure_dst and release_dst, which is not what I was trying to suggest.

`tile_regs_acquire()` and `tile_regs_commit()` should go around the math portion, while `tile_regs_wait()` and `tile_regs_release()` should go around the pack portion of the compute kernel.

An example of where these should be placed is in `tt_metal/kernels/compute/eltwise_binary.cpp`